docs/proposals/vertical-pod-autoscaler.md

```diff
@@ -0,0 +1,75 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+<strong>
+The latest 1.0.x release of this document can be found
+[here](http://releases.k8s.io/release-1.0/docs/proposals/vertical-pod-autoscaler.md).
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+## Abstract
+
+Vertical Pod Autoscaler is a feature that periodically adjust pods' [Resources](resource-qos.md#resource-specifications) based on demand signals.
+This is a natural extension of [Initial Resources](initial-resources.md) plugin.
+
+**Note:** since there is no plan to implement the feature immediately (as for 9/9/15) this document is just an overview how it should be done
+rather than detailed design.
+
+## Motivation
```

Comment:
You might want to mention that VPA is targeted at relatively long time-scale adjustments based on (in the fullness of time) potentially complex data mining of historical behavior. In particular that it is not intended to fill the same role as Borg/Mesos resource estimation.